### ISSUE TITLE
Non zero exit failure

### ISSUE THREAD

**Issue by carlskii** [joefitzgerald/packer-windows#177]
Not sure whats happening here, but my build is failing with this:
```
    vmware-iso: SDelete is set for 1 pass.
    vmware-iso: Free space cleaned on C:\
    vmware-iso: 1 drives zapped
    vmware-iso:
==> vmware-iso: Provisioning with shell script: /var/folders/b4/grsjj0d52t33m9jvc6g8_5440000gn/T/packer-shell510891355
    vmware-iso: rm: cannot remove '/tmp/vmware-vmsvc.log': Device or resource busy
    vmware-iso: rm: cannot remove '/tmp/vmware-vmusr.log': Device or resource busy
==> vmware-iso: Stopping virtual machine...
==> vmware-iso: Deleting output directory...
Build 'vmware-iso' errored: Script exited with non-zero exit status: 1

==> Some builds didn't complete successfully and had errors:
--> vmware-iso: Script exited with non-zero exit status: 1
```

**Comment by kensykora** [comment]
What version of vmware fusion are you using?

**Comment by carlskii** [comment]
Professional Version 8.0.0 (2985594)

**Comment by kensykora** [comment]
hrm, okay. I'm still running my builds on v7 and haven't yet upgraded to 8. You may want to try and post this issue on https://github.com/mitchellh/packer/issues since it may not be related to what we're doing with the box.

Which image were you trying to build? Did you make any modifications to it when you ran it?

**Comment by carlskii** [comment]
I'm using the windows_2012_r2 profile. Only modification I made was that I added my own powershell provisioner. If I remove:

```
{
      "type": "shell",
      "inline": [
        "rm -rf /tmp/*"
      ]
    }
```

It works OK.

**Comment by joefitzgerald** [comment]
I can reproduce this issue also.

**Comment by sneal** [comment]
Fixed by 975eac3ae2871bf2632bdc5eded09a958a948b48
